Dockerfile

```diff
@@ -1,41 +1,22 @@
-# WIP
-#
-# - how does this relate to the file in /tmp/build/ ?
-# - how should it relate to Make commands?
-#   - example(s):
-#     - https://github.com/openshift/cluster-image-registry-operator/blob/master/Makefile
-#   - possible commands:
-#     - make build  # build go app into binary
-#     - make build-image # build container image with included binary
-#     - make build-all # build binary + docker image?
-#     - make build-devel # different? or dangerous, to use alpine base?
-#
-# FROM openshift/origin-release:golang-1.10 as builder
-# WORKDIR /go/src/github.com/openshift/console-operator
-# COPY . .
-# RUN make build
-#
-#
 FROM openshift/origin-release:golang-1.10 as builder
-COPY . /go/src/github.com/openshift/console-operator/
-RUN cd /go/src/github.com/openshift/console-operator && \
-    go build ./cmd/console-operator
+WORKDIR /go/src/github.com/openshift/console-operator
+COPY . .
+RUN make build
 
 FROM centos:7
-# TODO: label for enabling install to pick this up:
-# LABEL io.openshift.release.operator true
+RUN useradd console-operator
+USER console-operator
+COPY --from=builder /go/src/github.com/openshift/console-operator/tmp/_output/bin/console-operator /usr/bin
 
-COPY --from=0 /go/src/github.com/openshift/console-operator /usr/bin/
+# these manifests are necessary for the installer
 COPY deploy/00-crd.yaml \
     deploy/01-namespace.yaml \
     deploy/02-rbac.yaml \
     deploy/03-operator.yaml \
     /manifests/
-# atm no assets
-# COPY tmp/build/assets /opt/openshift/
 
-RUN useradd console-operator
-USER console-operator
+# finally to enable install integration
+#LABEL io.openshift.release.operator true
 
-ENTRYPOINT []
-CMD ["/usr/bin/console-operator"]
\ No newline at end of file
+# entrypoint specified in 03-operator.yaml as `console-operator`
+CMD console-operator
```

Comment:
Better to make it an array with the full path as before

Reply:
Its now building in the root dir I believe, but I can still update to array.

Reply:
ah no i did put it back in `/usr/bin`.

Reply:
testing this now

Reply:
Quotes fixed.  Heh.